sw/host/opentitanlib/src/util/num_de.rs

```diff
@@ -237,4 +250,23 @@ mod test {
         assert_eq!(*data.hex, 119);
         Ok(())
     }
+
+    #[test]
+    fn byte_field_test() {
+        assert_eq!(Vec::from_str("0x1"), Ok(vec![0x1]));
+        assert_eq!(
+            Vec::from_str("0x4b4b4b4b4b4ba5a5"),
```

Comment:
nit: something like the following would make it clear that the checks are indeed correct:
```suggestion
            Vec::from_str("0x0706050403020100"),
```